\documentclass{article}

\usepackage[utf8]{inputenc}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage[letterpaper, portrait, margin=1in]{geometry}
\usepackage{gensymb}
\title{Molecules and Cells HW 7}
\date{October 13th, 2016}

\newcommand{\makefig}[2]{
\begin{figure}[h]
\centering
\includegraphics[scale=#1]{#2}
\end{figure}
}

\begin{document}

\maketitle

1a. reverse transcriptase\\
1b. retrovirus\\
1c. bacteriophage\\
DNA helicase - opens up DNA helix by breaking hydrogen bonds between nucleotides.\\
DNA polymerase - copies SNA strand in the 5' to 3' direction to make a new strand of DNA\\
isomerase- changes the structure of a molecule from one structure to another\\
exon - a section of DNA that is left after mRNA processing\\
intron - a section of DNA that is removed during mRNA processing\vspace{2mm}

2. Yes, this is unusual because the amounts of A and T are not the same and the amounts of G and C are not the same. This might be because the virus contains single strand DNA.
\vspace{2mm}

3a. 1 and 7 are brothers. 2 and 8 are brothers. 3 and 6 are brothers. 4 and 5 are brothers.
 
3b. Using the same process (hybridiztion probe) and comparing the results with the parents and the 4 sets to fin a match.
\vspace{2mm}

4. \makefig{0.5}{P4.png}
Adding phosphate groups would increase the size of the molecule and make it significantly more acidic, so we can choosing points that slope up and to the left. 
\vspace{2mm}

5. F. 10 cycles would give $2^{10}$-fold amplification, 20 cycles would give $2^{20}$-fold and so on
\vspace{2mm}

6. Since each nucleotide has a probability of $1/4$ of occuring, we can find the probability of the given sequence is
$$\frac{1}{4^n}$$
where n is the number of specific nucleotides in the sequence.

Therefore, if $X$ is the total number of nucleotides,
$$\frac{X}{4^n}$$
is the number of cuts and
$$\frac{X}{\frac{X}{4^n}}=4^n$$
is the average length of the DNA fragment.

Plugging in the various n values for the enzymes, Not I has an average length of 66 kb, which is close to the given value of 70 kb.

For the other enzymes, the value is not as close as for Not I.
\vspace{2mm}

7a. Genomic DNA library. The promoter would not be transcribed over to the RNA, so it would not be in the cDNA library\\
7b. cDNA library. The tRNA  is derived from the mRNA, cDNA library should be used.\\
7c. cDNA library. The products must be translated from the RNA, so it makes sense to look at the cDNA library\\
7d. Genomic DNA library. This will show the full sequence, which will contain A and the genes surrounding it
7e. Genomic DNA library. The full DNA region would contain promoters and operators that would increase protein production. 
\vspace{2mm}

8. To transport genetic information from one bacterium to another.

8b. An origin of replication - where replication starts
polylinkers - a short stretch of nucleotides which has ten to twenty restriction enzyme sites which help insert a gene and take it out
selection marker - Something to make it easy to distinguish which bacteria have the recombinant DNA. Eg. - Amp

8c. \makefig{1}{P8.png}

9. The third cycle. During the first cycle, the newly synthesized DNA will have the correct ending on one side. During the second cycle, strands with the correct ending on opposite sides will work together to make a single strand with the correct length, but not a double-sided strand. Only in the third cycle is the single strand of the correct length used to synthesize a double-sided strand of correct length.

10. (1) and (8). These primers correspond to the ends of the relevant gene sequence. The primer for the top is the inverse of the right side, while the primer for the bottom is the inverse of the left side.
\vspace{2mm}

11. MicroRNAs are noncoding RNAs that are incorporated into a protein
complex called \textbf{RISC}, which searches the \textbf{mRNA} in the cytoplasm for sequence complementary to that of the miRNA. When such a molecule is found, it is then targeted for \textbf{destruction}. RNAi is triggered by the presence of foreign \textbf{double-stranded RNA} molecules, which are digested by the \textbf{dicer} enzyme into shorter fragments approximately 23 nucleotide pairs in length.
\vspace{2mm}

12. A
\vspace{2mm}

13. The probe should be the operator region the protein binds to regularly.
\makefig{0.7}{P13.png}
\vspace{2mm}

14. C. Since the activator was removed, that section of DNA is not displayed on the positive signal side.

15a. Genomic because the genomic library would contain the proper restriction enzyme sites , which would allow it to be easily integrated into a plasmid.

15b. Taq polymerase or other DNA polymerases

15c. Reverse transcriptase

16a. The Rsal-B gene codes of red pigment. 8-10 were missing that pigment.\\
16b. Rsal-A codes for the green pigment. Males 5-7 were missing that.\\
16c. Males 1-4 had normal vision.
\end{document}